### ISSUE TITLE
Orbit lines vibrate and sometimes flip

### ISSUE THREAD

**Issue by xen-42** [PacificEngine/OW_CommonResources#10]
Lines move much more compared to base game. Sometimes the line will flip so that instead of fading in the retrograde direction it fades in the prograde direction

**Comment by salomj** [comment]
Yeah, The jiggling has to do with imprecision with floating point numbers, and only seems to impact larger orbits. I see a few different ways I can fix the jiggling. 

I am assuming the line flipping has to do with the New Horizon's mod? Do you have a link where you seeing it?

**Comment by xen-42** [comment]
Weird that the jittering doesn't happen in the base game but then happens for the same planets with the modifed orbit lines in common resources.

It's hard to get a picture of it since it happens for about a single frame. I just saw it on the orbitline for the orbital probe cannon so I doubt its New Horizons. Maybe something to do with how the current angle is calculated, since it seems to happen about once per orbit so maybe when it goes from 360 to 0 degrees. If you look at the cannon you should see it happen within maybe 30 seconds.

**Comment by salomj** [comment]
Hmm... I wonder if this is only happening with Linear Orbits then, and if it happens with exponential orbits at all.
